lib/vm-to-zones.js

```diff
@@ -151,16 +154,37 @@ function isNetOwned(vm, netw) {
 	return ((netw.owner_uuids || []).indexOf(vm.owner.uuid) !== -1);
 }
 
+function isProxied(ent, config) {
+	var zoneConfig = config.forward_zones[ent.zone];
+	if (!zoneConfig.proxy_networks)
```

Comment:
Same braces thing than above. Also, why parentheses around the boolean keywords?